.eslintignore

```diff
@@ -0,0 +1,3 @@
+.idea/
```

Comment:
Text editor dirs should be in user-specific configs

Reply:
What do you mean by «user-specific configs»?

Reply:
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
It's interesting, thanks! But it is not `.gitignore`, it is `.eslintignore` file.

Reply:
Do we have JS files in that directories?